docs/user-guide/get-started.md

```diff
@@ -39,7 +41,11 @@ If you use a pretty printer alongside Stylelint, you should turn off any conflic
 
 ## Linting everything else
 
-You'll need to use a [PostCSS syntax](https://github.com/postcss/postcss#syntaxes). We recommend [extending](../user-guide/configure.md#extends) a shared config that includes the appropriate syntax for your preferred language or library. For example, you can extend the [stylelint-config-standard-scss shared config](https://www.npmjs.com/package/stylelint-config-standard-scss) to lint SCSS.
+You'll need to use a [custom syntax](usage/options.md#customsyntax) written by the community.
```

Comment:
Use "custom syntax" language rather than introducing PostCSS syntax at this stage.